client/src/main/java/io/pravega/client/tables/impl/KeyFamilySerializer.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.tables.impl;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import io.pravega.client.stream.Serializer;
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import javax.annotation.Nullable;
+import org.apache.commons.lang3.SerializationException;
+
+/**
+ * Serializer for Key-Value Table Key Families.
+ */
+public class KeyFamilySerializer implements Serializer<String> {
+    @VisibleForTesting
+    static final int MAX_KEY_FAMILY_LENGTH = 8191; // It can't be longer than TableSegment.MAX_KEY_LENGTH
```

Comment:
Good catch. I actually wanted to make it more restrictive, but I forgot. I restricted this to 1KB for now. It will always be easier to increase the limit than shrink it in the future.